.github/workflows/ansible-linting-chek.yml

```diff
@@ -0,0 +1,20 @@
+name: Ansible Lint check
```

Comment:
There's a typo in the filename (chek should be check), could you please rename it?

Reply:
thanks. I renamed it.

Typo-man got me here /o\